Gemfile.lock

```diff
@@ -330,7 +330,6 @@ GEM
     matrix (0.4.2)
     method_source (1.1.0)
     mini_mime (1.1.5)
-    mini_portile2 (2.8.7)
```

Comment:
This is weird... Do you know why is being removed?

Reply:
I have no idea 🤷🏻‍♀️ both the `ruby` and this line were removed when I ran `bundle _2.5.14_ install`

Reply:
After adding back the platform, this gem was added back to my Gemfile.lock file. I think it's still a dependency but it's not needed on your local platform. We have to make sure we keep it.